### ISSUE TITLE
Quest: Warchief's Blessing

### ISSUE THREAD

**Issue by underlord-oberon** [plantain-gnocchis/wow-voiceover-tbc-wotlk#75]
Add-on version: AI_VoiceOver_v1.4.5
Data version: AI_VoiceOverData_VanillaExtra_v1.4

Eastern Plaguelands / Acherus: The Ebon Hold / NPC: Highlord Darion Mograine / Quest: Warchief's Blessing
Link: https://www.wowhead.com/wotlk/quest=13189/warchiefs-blessing

Description: playback ok.

Progress: no playback.

Completion:  no playback.

**Comment by carevass** [comment]
This should be fixed in AI_VoiceOverData_WoTLK_v1.1.
